x-pack/plugins/security_solution/public/detection_engine/rule_response_actions/endpoint/field_name.tsx

```diff
@@ -128,13 +128,14 @@ const FieldNameFieldComponent = ({
                 noSuggestions={false}
                 options={optionsAsComboBoxOptions}
                 fullWidth
+                isClearable={false}
                 selectedOptions={value && valueInList ? [{ value, label: value }] : undefined}
                 onChange={(newValue) => {
                   if (newValue.length === 0) {
-                    // Don't allow clearing the type. One must always be selected
-                    return;
+                    setValue('');
```

Comment:
Is it possible to add a test for this issue?

Reply:
👍  I'll add this to our cypress tests

Reply:
I converted the PR to draft again, not to make too much noise. Thanks @paul-tavares !